versionhistory.md

```diff
@@ -32,47 +38,28 @@ Status codes:
 * 200 OK - if the version history exists and is accessible by the current user
 * 401 Unauthorized - if you are not authenticated and versioning is not public
 * 403 Forbidden - if you are not logged in with sufficient permissions and versioning is not public
-* 404 Not found - if the version history doesn't exist
+* 404 Not found - if you have the permission to review the version history and it doesn't exist
 
 ## Linked entities
 
-### Versions
+### Oldest Version
 
-**GET /api/versioning/versionhistories/<:versionHistoryId>/versions**
+**GET /api/versioning/versionhistories/<:versionHistoryId>/oldestversion**
 
-Retrieve a pageable list of versions for the provided version history identifier.  
-The versions are ordered by version number descending.
+Retrieve the oldest version in the history 
 
-```json
-{
-  "_embedded": {
-    "versions": [
-      {
-          "id": "102",
-          "version": "2",
-          "type": "version",
-          "created": "2019-10-31T09:44:46.617",
-          "summary": "Author order"
-      },
-      {
-        "id": "101",
-        "version": "1",
-        "type": "version",
-        "created": "2015-11-03T09:44:46.617",
-        "summary": "Fixing some typos in the abstract"
-      }
-    ],
-    "_links": {
-     "self": {
-       "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistories/1/versions"
-     }
-    },
-    "page": {
-     "size": 20,
-     "totalElements": 2,
-     "totalPages": 1,
-     "number": 0
-    }
-  }
-}
-```
+### Current Version
+
+**GET /api/versioning/versionhistories/<:versionHistoryId>/currentversion**
```

Comment:
The wording here compared to last version can be confusing
It may be better to use e.g. `current live version`

Reply:
I'd recommend keeping this as `currentversion`, but renaming `lastversion` (see below)